test/parallel/test-dgram-send-callback-buffer-length-empty-address.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const dgram = require('dgram');
+const client = dgram.createSocket('udp4');
+
+const buf = Buffer.allocUnsafe(256);
```

Comment:
The test doesn't rely on sending unspecified data, does it? Can you just use `const buf = Buffer.alloc(256, 'x');` as in the other test?